addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,153 +17,242 @@
-                           t-options='{"widget": "image", "qweb_img_responsive": False, "class": "o_image_128_max"}'
-                        ></a>
-                        <div class="flex-grow-1 o_partner_body" style="min-height: 64px;">
-                            <a t-attf-href="/partners/#{slug(partner)}?#{current_grade and 'grade_id=%s&amp;' % current_grade.id}#{current_country and 'country_id=%s' % current_country.id}">
-                                <span t-field="partner.display_name"/>
-                            </a>
-                            <div t-field="partner.website_short_description"/>
-                            <t t-if="any(p.website_published for p in partner.implemented_partner_ids)">
-                                <small><a t-attf-href="/partners/#{slug(partner)}#right_column">
-                                    <t t-esc="partner.implemented_partner_count"/> reference(s)
-                                </a></small>
-                            </t>
+        </div>
+        <!-- Off canvas filters on mobile-->
+        <div id="o_wcrm_offcanvas" class="o_website_offcanvas offcanvas offcanvas-end d-lg-none p-0 overflow-visible">
+            <div class="offcanvas-header">
+                <h5 class="offcanvas-title">Filters</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"/>
+            </div>
+            <div class="offcanvas-body p-0">
+                <div class="accordion accordion-flush">
+                    <div class="accordion-item">
+                        <h2 class="accordion-header">
+                            <button class="accordion-button fw-bold collapsed"
+                                type="button"
+                                data-bs-toggle="collapse"
+                                data-bs-target=".o_wcrm_offcanvas_grade"
+                                aria-expanded="false"
+                                aria-controls="o_wcrm_offcanvas_grade">
+                                Filter by category
+                            </button>
+                        </h2>
+                        <div class="o_wcrm_offcanvas_grade accordion-collapse collapse">
+                            <div class="accordion-body pt-0">
+                                <ul class="list-group list-group-flush">
+                                    <t t-foreach="grades" t-as="grade">
+                                        <li class="list-group-item d-flex justify-content-between align-items-center ps-0 pb-0 border-0">
+                                            <a t-attf-href="/partners#{ grade['grade_id'][0] and '/grade/%s' % grade['grade_id'][0] or '' }#{ current_country and '/country/%s' % slug(current_country) or '' }#{ '?' + (search_path or '') + '&amp;' + keep_query('country_all') }"
+                                            t-attf-class="#{grade['active'] and ' active' or ''} p-0 text-body" aria-label="See categories filters">
```

Comment:
```suggestion
                                            t-attf-class="#{grade['active'] and ' active' or ''} p-0 text-reset" aria-label="See categories filters">
```
It will simplify things in case we change the offcanvas colours